api/userTermsAndConditions/index.ts

```diff
@@ -0,0 +1,63 @@
+import * as express from 'express'
+import { config } from '../config'
+import { application } from '../config/application.config'
+import { GetUserAcceptTandCResponse, PostUserAcceptTandCResponse } from '../interface/userAcceptTandCResponse'
+import { http } from '../lib/http'
+import { isUserTandCPostSuccessful } from '../lib/util'
+import { getUserTermsAndConditionsUrl, postUserTermsAndConditionsUrl } from './userTermsAndConditionsUtil'
+
+export async function getUserTermsAndConditions(req: express.Request, res: express.Response) {
+    let errReport: any
+    if (!req.params.userId) {
+        errReport = {
+            apiError: 'UserId is missing',
```

Comment:
this error should be managed through common error method

Reply:
Most errors are different, so no point in handling them the same. Also, different endpoints will want to return different errors based on different conditions.